app/backend/src/couchers/models/users.py

```diff
@@ -405,12 +404,26 @@ class User(Base):
 
     @hybrid_property
     def has_completed_profile(self) -> bool:
-        return self.avatar_key is not None and self.about_me is not None and len(self.about_me) >= 150
+        # Check if user has at least one photo in their profile gallery
+        if self.profile_gallery_id is None:
+            has_photo = False
+        else:
+            session = object_session(self)
+            has_photo = session.query(
+                sa_select(1).where(PhotoGalleryItem.gallery_id == self.profile_gallery_id).exists()
+            ).scalar()
+
+        return has_photo and self.about_me is not None and len(self.about_me) >= 150
```

Comment:
IMO this is getting too complex for a model method. Should we move it into a function (e.g., in `couchers.sql`)?